palantirnet/st-bernards
# MSSB-43: Event calendar contrast

## JIRA Ticket(s)

- See ticket https://palantir.atlassian.net/browse/MSSB-43

## Description

- Branch from https://github.com/palantirnet/st-bernards/pull/6
- Changed button text color
![Screen Shot 2020-01-16 at 3 51 31 PM](https://user-images.githubusercontent.com/10488517/72572991-f298da00-3889-11ea-9436-ffabc682a14a.png)
- Changed colors of the events to pass WCAG standards
(examples:)
![Screen Shot 2020-01-16 at 4 29 24 PM](https://user-images.githubusercontent.com/10488517/72573049-2116b500-388a-11ea-861e-e8d642378a64.png)
![Screen Shot 2020-01-16 at 4 33 11 PM](https://user-images.githubusercontent.com/10488517/72573052-2411a580-388a-11ea-976f-cc05f7867ace.png)

## Testing instructions

Before - 
<img width="378" alt="Screen Shot 2020-01-16 at 6 03 05 PM" src="https://user-images.githubusercontent.com/10488517/72573213-823e8880-388a-11ea-80f1-38631ad23a1a.png">
After - 
<img width="372" alt="Screen Shot 2020-01-16 at 6 02 47 PM" src="https://user-images.githubusercontent.com/10488517/72573221-87033c80-388a-11ea-8948-2ba34dbc5941.png">
(two errors on hidden elements)

- Navigate to `cd web/themes/custom/blrfurther_d8`, run `gulp sass` to compile CSS.
- `drush cim`
- `drush cr`
- Visit http://st-bernards.local/events and make sure the colors pass a11y standards.
